model-optimizer/mo/ops/tdnncomponent.py

```diff
@@ -0,0 +1,35 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import numpy as np
+
+from mo.front.common.partial_infer.elemental import copy_shape_infer
+from mo.graph.graph import Graph, Node
+from mo.ops.op import Op
+from mo.utils.error import Error
+
+
+class TdnnComponent(Op):
+    op = 'tdnncomponent'
+    enabled = True
```

Comment:
`enabled = True` -> `enabled = False` because the operation should not work as extractor.

Reply:
Agreed. Thank for noting this.

Reply:
Still not changed.

Reply:
Sorry for that. Changed.